lib/routes.js

```diff
@@ -63,6 +64,7 @@ app.use(filterOldRequests)
 app.use('/poll', pollingRoutes)
 app.use('/terms_conditions', termsAndConditionsRoutes)
 app.use('/state', stateRoutes)
+app.use('/notify', cashboxRoutes)
```

Comment:
Maybe we can start a cashbox subroute. This can be /cashbox, and the notifyRemoval /removal.